packages/@aws-cdk/aws-ec2/lib/vpc.ts

```diff
@@ -520,12 +520,15 @@ export class VpcSubnet extends VpcSubnetRef implements cdk.ITaggable {
     });
   }
 
-  protected addDefaultRouteToIGW(gatewayId: string) {
-    new cloudformation.RouteResource(this, `DefaultRoute`, {
+  protected addDefaultRouteToIGW(
```

Comment:
Could you please add some jsdocs here and also mention that a dependency will be created?

Reply:
Done